src/much-selector.js

```diff
@@ -256,6 +266,22 @@ class MuchSelector extends HTMLElement {
     this._app.ports.disableChangedReceiver.send(this._disabled);
   }
 
+  get maxdropdownitems() {
+    return this._maxdropdownitems;
```

Comment:
You should initialize `this._maxdropdownitems` in the constructor. And it should probably be camel case.